go/cache/lru_cache_test.go

```diff
@@ -44,14 +45,12 @@ func TestSetInsertsValue(t *testing.T) {
 	cache.Set(key, data)
 
 	v, ok := cache.Get(key)
-	if !ok || v != data {
-		t.Errorf("Cache has incorrect value: %v != %v", data, v)
-	}
+	assert.True(t, ok)
+	assert.EqualValues(t, v, data)
 
 	values := cache.Items()
-	if len(values) != 1 || values[0].Key != key {
-		t.Errorf("Cache.Values() returned incorrect values: %v", values)
-	}
+	require.NotEmpty(t, values)
```

Comment:
This should be `assert` and not `required`. And it should test for exactly one element.
```suggestion
	require.Len(t, values, 1)
```